File: programme/feet-to-inches/feet-to-inches.js
// Write a programme to convert feet to inches
// Execute: node inches-to-feet.js

"use strict"

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Input  : ", (inches) => {
	const result = parseInt(inches) * 12;
	if (!isNaN(result)) {
		console.log(`Output : ${result}`)
	} else {
		console.log("An Error occured");
	}
	rl.close();
});

File: programme/feet-to-inches/FeetToInches.cs
// Write a programme to convert feet to inches
using System;

namespace UnitConversion
{
    public class FeetToInches
    {
        public static double ConvertFeetToInches(double feet) => feet * 12.0;

        public static void Main()
        {
            Console.Write("Input  : ");

            double feet;
            if (double.TryParse(Console.ReadLine(), out feet))
            {
                double inches = ConvertFeetToInches(feet);
                Console.WriteLine("Output : " + Math.Round(inches, 2));
            }
            else
            {
                Console.WriteLine("Invalid Input");
            }
        }
    }
}
